src/app/component/MissionCard.jsx

```diff
@@ -89,6 +105,16 @@ const MissionCard = withStyles(styles)(({ children, classes, mission, ...rest })
   );
 });
 
+MissionCard.defaultProps = {
+  mission: {
+    title: "No title supplied",
+    status: "",
+    pickUpLocation: {
+      address: "No Data",
+    },
+  },
+};
```

Comment:
This shape doesn't match what's being destructured from props.

The code sample i pasted was just a sample and I didn't write out the entire props object.

Reply:
I'm sorry @jwu910 , I'm not clear on what you want me to do.  "address" is a field of the Location object, per the [schma](https://github.com/factn/resilience-app/blob/master/src/app/model/schema.ts#L5) .  Do you want me to populate default values for all fields that are part of a mission?

Reply:
Default props aren't necessary, but its a feature that allows us to pre-define values for props going into the components.

Looking again, i think I misread something. The shape looks right, but if we're going to set default values for `title` and `address`, would we need default values for the rest of the properties in the missions prop?

Reply:
@jwu910 , I think that is a question that is above my paygrade!  :)  I would say, as an outsider looking in, I don't think defaultProps are needed at all, in as far as all these fields should be populated with something otherwise it wouldn't make sense to a volunteer (or a recipient).  @azmainamin may have some thoughts as well.

Reply:
You seem to be doing just fine lol.

What I was trying to get at was that PropTypes can be used at the component level to determine the data going in. If that data is coming from the backend, and its missing, thats a whole different problem. If its required on the backend, we can expect it on the front, the propTypes are here to help let the component know when props should be required.

Maybe going down a rabbit hole now:

Say I write a component:
```
const personDetail = (props) => {
  return (
    <div>
      {props.people.map(person => {
        return <div style={{color: props.color}}>
          {person.name}
          {person.age}
        </div>
      })}
    </div>
}
```

And you as another developer will be using `personDetail` in your component, you may write out:

```
<personDetail color='blue' />

// or

<personDetail people={[{name: 'Dave'}]} color='red' />
```

And because the component does not check to see if any values are missing when it's invoked, you're gonna throw a reference error(?) for people not a property of props, or age is not a property of person.

PropTypes and defaultProps(if we want them) help avoid some of these issues by giving the component a meaningful error message something like "`people is a required prop of personDetail, but none was provided`"

I think you're right on us not needing them per se, would be a whole different issue if they were missing.

Reply:
What Josh said. I am not the biggest fan of default props. Its very easy to hide a bug (BE not sending the correct data) with a default Prop. I would use them if not having the data will break some logic: `const myVar = obj.prop` will break if obj is undefined. 

In this case, I have no problem not using default props.

Reply:
@jwu910 @azmainamin , ok I have removed the default props.  I have also added in some more card content, in the interest of moving these features along.  Let me know what you think.